lib/pbkit/pbkit.h

```diff
@@ -144,6 +144,10 @@ void pb_create_gr_ctx(   int ChannelID,
                       struct s_CtxDma *pGrObject  );
 void pb_bind_channel(struct s_CtxDma *pCtxDmaObject);
 
+uint8_t *pb_depth_stencil_buffer();
+DWORD pb_depth_stencil_pitch();
+DWORD pb_depth_stencil_size();
```

Comment:
I always thought pbkit had this interface backwards. Currently pbkit is a mix of a low-level and high-level driver.

pbkit can submit pushbuffers, but it also constructs them, some using rather high level operations.
Instead, I believe that pbkit should allow submission of pushbuffers, but it should primarily be the users responsibility to construct them.
pbkit should only control the hardware so it can process commands from the pushbuffers.

More importantly, pbkit also manages buffers internally (sometimes like a singleton), but doesn't allow the user to manually create or control these buffers.
Instead, I believe that pbkit should provide the user with buffer creation / managing functions, but the user should always explicitly create these buffers and keep track of them.

This means I'd prefer to see something like `pb_create_depth_stencil_buffer(size, pitch)` and then the user should be responsible for creating these buffers.
So, in an ideal world, the application would never even have to query for the buffer address / pitch / size, because it created the buffer itself.

This is also required (or at least helpful) for more advanced applications which want to switch render-targets.